packages/cli/src/handlers/compile.ts

```diff
@@ -56,8 +61,11 @@ export const compile = async (options: GenerateHandlerOptions) => {
     );
     return explores;
 };
-export const compileHandler = async (options: GenerateHandlerOptions) => {
-    const explores = await compile(options);
+export const compileHandler = async (
+    options: GenerateHandlerOptions,
+    command: any,
+) => {
+    const explores = await compile(options, command);
```

Comment:
Could we add some logging so the user knows that dbt compile is running and expected 

Also how do logs from dbt look? and errors?

Reply:
I tried to have a spinner but there is already an active spinner for preview/deploy and causes the terminal to flick between the 2.  

I can enable the dbt logs if it does't cause the same issue as above.

Reply:
Yeah I think showing logs is great - just pipe it so it looks identical to if they had run `dbt compile` themselves